IDE/Espressif/ESP-IDF/examples/wolfssl_benchmark/main/main.c

```diff
@@ -24,21 +24,23 @@
 #include "sdkconfig.h"
 
 /* wolfSSL */
-#include <wolfssl/wolfcrypt/settings.h>
-#include <user_settings.h>
+#include <wolfssl/wolfcrypt/settings.h> /* references user_settings.h */
+/* Do not explicitly include wolfSSL user_settings.h */
```

Comment:
Combine comment above into this one.

Reply:
I've updated the comment section regarding `user_settings.h` and `settings.h`.

This will be a template for all future files, so extra attention and any additional feedback definitely welcome.

Reply:
For reference, I have another wording for the `user_Settings.h` and `settings.h` comment to use, instead:

```c
/* wolfSSL */
/* Always include wolfcrypt/settings.h before any other wolfSSL file.    */
/* Reminder: settings.h pulls in user_settings.h; don't include it here. */
#ifdef WOLFSSL_USER_SETTINGS
    #include <wolfssl/wolfcrypt/settings.h>
#else
    #error "Missing WOLFSSL_USER_SETTINGS in CMakeLists or Makefile: \
CFLAGS +=-DWOLFSSL_USER_SETTINGS"
#endif
```
This is an important point, as it is a common stumbling block for new users. I'd like for it to be consistent everywhere, so I'll be updating examples as I go.